src/array/cuda/uvm/array_index_select_uvm.cuh

```diff
@@ -34,9 +35,10 @@ __global__ void IndexSelectMultiKernelAligned(
         ((uint64_t)(&array[in_row * num_feat]) % CACHE_LINE_SIZE) /
         sizeof(DType);
     col = col - idx_offset;
+    const auto out_row_ = perm ? perm[out_row] : out_row;
```

Comment:
Can we differentiate more between `out_row` and `out_row_`? Maybe `out_row_index` and `out_row` respectively? In many places in dgl a variable suffixed with `_` denotes a member variable.